server/controllers/applications.coffee

```diff
@@ -3,10 +3,13 @@ async = require 'async'
 log = require('printit')()
 exec = require('child_process').exec
 
-updateController = (callback) ->
-    exec "npm -g update cozy-controller", (err, stdout) ->
-        if err
-            callback err
+updateController = (callback, count=0) ->
```

Comment:
callback should be the last parameter. Test count type to know, if the callback has been given as first argument.